src/dcat-us/compile-dcat-feed.ts

```diff
@@ -25,6 +22,32 @@ function removeUninterpolatedDistributions(distributions: any[]) {
   return distributions.filter((distro) => !(typeof distro === 'string' && distro.match(/{{.+}}/)?.length));
 }
 
+function generateDcatItem(feedTemplate, feedTemplateTransforms, geojsonFeature) {
+  const defaultFields = {
+    '@type': 'dcat:Dataset'
+  };
+
+  // moving feature properties fields at the top level
+  // and geometry to second as the values for the properties
+  // can be easily referenced in template
+  const dcatFeedData = { 
+    ...geojsonFeature?.properties, 
```

Comment:
I think you don't need the `?` as geojson should always have `properties`.  But if you are worried about it not being defined, I think this code will cause an exception.  If `properties` is undefined, it will try to destructure `undefined` and you'll have an error.

I think this is probably not under test.  I think you should implement test coverage, and add tests for whatever is not covered.

Reply:
That is true. My intention was to not allow it to fail even if some if some of geojson attributes are missing.  I have removed the optional chaining operator for now and added test coverage for it.